rollup.examples.config.js

```diff
@@ -0,0 +1,18 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+import styles from 'rollup-plugin-styles';
+
+export const globals = id =>
```

Comment:
There's something I find a bit counter-intuitive about these base config files. For example, I tripped up here because I thought I could replace this line with an import like so:

```js
import { globals } from './rollup.src.config';
```

But that caused the build script in the examples to fail. That's because importing from that other Rollup config brought along its dependencies, which are not included in the package.json for each of the example packages. In other words, for example, `rollup-plugin-postcss` is not included as a dependency for any of the example-* packages, but is a dependency in the other packages.

Reply:
This is expected - a solution would be to define a base config that only uses common dep. But here this is probably not worth the trouble just for the `globals` definition.